src/metabase/automagic_dashboards/core.clj

```diff
@@ -969,50 +971,46 @@
        (fill-related max-related (related-selectors (-> root :entity type)))))
 
 (defn- filter-referenced-fields
-  "Return a map of fields referenced in filter cluase."
+  "Return a map of fields referenced in filter clause."
   [root filter-clause]
   (->> filter-clause
        filters/collect-field-references
-       (mapcat (fn [[_ & ids]]
-                 (for [id ids]
-                   [id (->field root id)])))
```

Comment:
This was the actual fix here, as you can see destructuring something like `[:field 1 {:temporal-unit :year}]` as `[_ & ids]` is busted -- this was what was breaking